frontend/src/components/header/header.tsx

```diff
@@ -82,7 +83,12 @@ export const Header = (): React.ReactElement => {
     },
     logout: {
       label: 'Logout',
-      onClick: () => navigate('#'),
+      onClick: (): void => {
+        logOut();
```

Comment:
If I'm not mistaken your `logOut` does completely nothing because we need to call `dispatch(logOut())`